<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_read">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_write">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
